validator/sawtooth_validator/concurrent/threadpool.py

```diff
@@ -41,13 +42,26 @@ def __init__(self, max_workers=None, name='', trace=None):
 
         self._workers_in_use = atomic.Counter()
 
-        self._max_workers = max_workers
-        if self._max_workers is None:
-            # This is the same default as ThreadPoolExecutor, but we want to
-            # know how many workers there are for logging
-            self._max_workers = multiprocessing.cpu_count() * 5
         super().__init__(max_workers)
 
+        if metrics_registry:
+            # Tracks how long tasks are queued
+            self._time_in_queue_gauge = GaugeWrapper(
+                metrics_registry.gauge(
+                    'threadpool.time_in_queue'))
```

Comment:
This needs the threadpool name prefixed to it, or else all of the threadpools will be aggregated into one metric.